doc/man1/flow.1

```diff
@@ -0,0 +1,364 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.8.
```

Comment:
Maybe flow - as the flagship executable - deserves a bit more prose in the man page?

Reply:
Printready recommendations for descriptions are welcome.